src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs

```diff
@@ -135,6 +141,73 @@ internal struct USER_INFO_1
             public string usri1_comment;
             public uint usri1_flags;
             public string usri1_script_path;
+
+#if NET7_0_OR_GREATER
+            [CustomMarshaller(typeof(USER_INFO_1), MarshalMode.Default, typeof(USER_INFO_1Marshaller))]
+            public static class USER_INFO_1Marshaller
```

Comment:
```suggestion
            public static class Marshaller
```

Reply:
Is there a constraint on Marshaller class naming to make it work ? or is it just a standard convention ?

Reply:
I just thought the name looked awkward and that `Marshaller` was the convention.